src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -74,7 +75,10 @@ void BasePlaylistFeature::initActions() {
             &BasePlaylistFeature::slotCreatePlaylist);
 
     m_pRenamePlaylistAction = new QAction(tr("Rename"), this);
-    m_pRenamePlaylistAction->setShortcut(kRenameSidebarItemShortcutKey);
+    m_pRenamePlaylistAction->setShortcuts(QList<QKeySequence>{
+            QKeySequence(kRenameSidebarItemShortcutKey),
+            QKeySequence(kRenameSidebarItemAlternativeShortcutKey)});
```

Comment:
```suggestion
```
If we want to show the short cut, we need to change it form F2 to the value which is picked in kbd.cfg.

Reply:
Updated the code to display the correct shortcut in the display menu (though it does not dynamically react to toggling keyboard shortcuts on/off at runtime, as that requires the infrastructure implemented in #13082).